app/models/like.rb

```diff
@@ -1,2 +1,9 @@
 class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :posts, class_name: 'Post'
```

Comment:
- [ ]  @David-Kasilia great work in your models, however, you don't need to specify the post class , also you used plural instead of `posts` it should be `post`.